### ISSUE TITLE
Honeymap not displaying anything

### ISSUE THREAD

**Issue by Norup123** [threatstream/mhn#709]
I have MHN set up, and have deployed a cowrie HP. The honeypot runs and i am connecting to it from a vpn in another country
Everything is running on the mhn server
![image](https://user-images.githubusercontent.com/54578187/64327986-f60f8100-cfcc-11e9-8462-10329c77b596.png)
Nothing shows up on the map at /ui/honeymap/ When im connecting to the cowrie. The attacks are showing up on the /ui/attacks page

**Comment by pouyiouk** [comment]
In reference to my issue #706, only conpot and shockpot events are populated to the map only. Deploy a conpot sensor to double confirm. Anything else does not update the map.

**Comment by Norup123** [comment]
How do i test the conpot?

**Comment by pouyiouk** [comment]
Yep, same problem as mine. You have loads of attacks, but only conpot and shockpot attacks are populated on the map. Other attacks like Suricata, pof, cowrie,amun attacks don't show on the map.

**Comment by neil-fox** [comment]
FYI Dionaea works fine for the map, just be sure to adjust the logging otherwise you generate gigs of logs in no time.

I would be interested to know if anybody has the flags working on the Attack Stats page. That has never worked for me.

**Comment by pouyiouk** [comment]
@neil-fox implement the solution provided in pull request #698. everything will be fixed regarding unknown flags.

**Comment by neil-fox** [comment]
@pouyiouk Thanks for the heads up. I've run the .sh file and restarted my MHN services but still not seeing the flags being resolved. Is there something obvious I've missed?

**Comment by Norup123** [comment]
the .sh file runs automatically when you install MHN.

Go to https://github.com/threatstream/mhn/pull/698/files
and copy paste that code in to your own `server/mhn/ui/utils.py`
Then restart MHN, and make sure all the services are running

**Comment by neil-fox** [comment]
@Norup123 My utils.py now matches that updated code but I now can't get all the mhn services to start:

```
import requests
from flask import current_app as app
from mhn.ui import constants
from config import MHN_SERVER_HOME
import os
from werkzeug.contrib.cache import SimpleCache
import socket
import struct
from mhn.api.models import Sensor
import geoip2.database

flag_cache = SimpleCache(threshold=1000, default_timeout=300)
sensor_cache = SimpleCache(threshold=1000, default_timeout=300)

geoip2_reader = geoip2.database.Reader(MHN_SERVER_HOME+'/../../GeoLite2-City.mmdb')

def is_RFC1918_addr(ip):
    # 10.0.0.0 = 167772160
    # 172.16.0.0 = 2886729728
    # 192.168.0.0 = 3232235520
    RFC1918_net_bits = ((167772160, 8), (2886729728, 12), (3232235520, 16))

    try:
        # ip to decimal
        ip = struct.unpack("!L", socket.inet_aton(ip))[0]

        for net, mask_bits in RFC1918_net_bits:
            ip_masked = ip & (2 ** 32 - 1 << (32 - mask_bits))
            if ip_masked == net:
                return True
    except Exception as e:
        print 'Error ({}) on is_RFC1918_addr: {}'.format(e, ip)

    return False


def get_flag_ip(ipaddr):
    if is_RFC1918_addr(ipaddr):
        return constants.DEFAULT_FLAG_URL

    flag = flag_cache.get(ipaddr)
    if not flag:
        flag = _get_flag_ip_localdb(ipaddr)
        flag_cache.set(ipaddr, flag)
    return flag

def get_sensor_name(sensor_id):
    sensor_name = sensor_cache.get(sensor_id)
    if not sensor_name:
        for s in Sensor.query:
            if s.uuid == sensor_id:
                sensor_name = s.hostname
                sensor_cache.set(sensor_id, sensor_name)
                break
    print 'Name: %s' % sensor_name
    return sensor_name

def _get_flag_ip_localdb(ipaddr):
    flag_path = '/static/img/flags-iso/shiny/64/{}.png'
    try:
        r = geoip2_reader.city(ipaddr)
        ccode = r.country.iso_code
    except Exception:
        app.logger.warning("Could not determine flag for ip (LOCALDB): {}".format(ipaddr))
        return constants.DEFAULT_FLAG_URL
    else:
        # Constructs the flag source using country code
        flag = flag_path.format(ccode.upper())
        if os.path.exists(MHN_SERVER_HOME +"/mhn"+flag):
            return flag
        else:
            return constants.DEFAULT_FLAG_URL

def _get_flag_ip(ipaddr):
    """
    Returns an static address where the flag is located.
```

MHN Services:

geoloc                           RUNNING   pid 24133, uptime 0:12:03
honeymap                         RUNNING   pid 24131, uptime 0:12:03
hpfeeds-broker                   RUNNING   pid 24142, uptime 0:12:03
mhn-celery-beat                  FATAL     Exited too quickly (process log may have details)
mhn-celery-worker                FATAL     Exited too quickly (process log may have details)
mhn-collector                    RUNNING   pid 24141, uptime 0:12:03
mhn-uwsgi                        RUNNING   pid 24137, uptime 0:12:03
mnemosyne                        RUNNING   pid 24132, uptime 0:12:03

**Comment by Norup123** [comment]
After restarting the servers, my conpot is also no longer showing on the map. Getting lots of attacks on the attacks page on both cowrie and conpot still, but nothing show on the map anymore

**Comment by Norup123** [comment]
@neil-fox try `chown www-data /var/log/mhn/mhn.log`
and then `sudo supervisorctl start mhn-celery-worker` and the other one
Can you get conpot to show up on your map?
Mine has completely stopped working. 

It says "connection to back-end established, All services are running, and i get conpot attacks on my attack page, but the map is emtpy

**Comment by neil-fox** [comment]
@Norup123 Only just seen your message. I've tried what you suggested but it still doesn't work for me so not sure what i'm missing.

I don't have conpot deployed but my dionaea attacks have stopped showing on my attack map which is annoying. Port 445 is now showing as filtered on my nmap scan but i haven't made any firewall changes for that port.

**Comment by Norup123** [comment]
Looks like the same issue i am having. port 445 comes up as filtered, so no attacks are really coming in. Im confused about whats really going on, because as you say, it just randomly stops working
